README.md

```diff
@@ -10,6 +10,7 @@
 
 Bats is a [TAP](https://testanything.org/)-compliant testing framework for Bash.  It provides a simple
 way to verify that the UNIX programs you write behave as expected.
+Bats itself is also written in Bash and requires Bash 3.2 or above.
```

Comment:
Maybe we should just bot "3.2 or above" into the first sentence on line 11? Anyways, did you see https://bats-core.readthedocs.io/en/stable/support-matrix.html?

Reply:
> Maybe we should just bot "3.2 or above" into the first sentence on line 11?

I'll update it in that way.

> did you see https://bats-core.readthedocs.io/en/stable/support-matrix.html?

I'm sorry. I tried to find any version information about Bash using `git grep version`, but I missed it. (I again tried to look it up in the result of `git grep version`, but I couldn't find it until I rerun it as `git grep 'Bash version'`...)